crates/stages/src/stages/index_account_history.rs

```diff
@@ -38,12 +43,20 @@ impl<DB: Database> Stage<DB> for IndexAccountHistoryStage {
     async fn execute(
         &mut self,
         provider: &DatabaseProviderRW<'_, &DB>,
-        input: ExecInput,
+        mut input: ExecInput,
```

Comment:
we're modifying the copy of the `Input` and not the mutable reference, so it should be fine